README.md

```diff
@@ -1,116 +1,179 @@
-  - [Live-Demo](#live-demo)
-- [:busts_in_silhouette: Authors](#authors)
-- [:telescope: Future Features](#future-features)
-- [:handshake: Contributing](#contributing)
-- [:star:️ Show your support](#support)
-- [:pray: Acknowledgements](#acknowledgements)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
 
-# :book: [Awesome Books]
+# 📖 [To Do List] <a name="about-project"></a>
+
+**[To Do List]** A simple web application that allows you to create and manage a to-do list. You can add tasks, mark them as completed, reorder them, and clear completed tasks.
+
+## 🛠 Built With <a name="built-with"></a>
+
+- Plain JavaScript
 
-**[To-Do List]** is a microverse project where we build a basic website that allows users to add/remove list. We achieved that by using JavaScript objects and arrays. We also dynamically modified the DOM and add basic events.
+### Tech Stack <a name="tech-stack"></a>
 
+<ul>
+  <li>HTML5</li>
+  <li>CSS3</li>
+  <li>Linters</li>
+  <li>JavaScript</li>
+  <li>Webpack</li>
+</ul>
 
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://github.com/microverseinc/linters-config">Linters</a></li>
+    <li><a href="https://github.com/microverseinc/linters-config/tree/master/html-css-js">HTML</a></li>
+    <li><a href="https://github.com/microverseinc/linters-config/tree/master/html-css-js">CSS</a></li>
+    <li><a href="https://github.com/microverseinc/linters-config/tree/master/html-css-js">JS</a></li>
+    <li><a href="https://webpack.js.org/guides/output-management/#setting-up-htmlwebpackplugin">webpack</a></li>
+  </ul>
+</details>
 
-## :hammer_and_wrench: Built With
+<!-- Features -->
 
-### Tech Stack
+### Key Features <a name="key-features"></a>
 
-**Client**
-HTML, CSS and Javascript
+The To Do List website provides the following features:
 
-### Key Features
+- Add tasks to your to-do list.
+- Mark tasks as completed by checking the checkbox.
+- Reorder tasks by dragging and dropping.
+- Clear all completed tasks with the click of a button.
 
-- **[- Use Javascript, HTML & CSS linters]**
-- **[- Build a basic website that allows users to add/remove task list.]**
-- **[- Used JavaScript objects and arrays.]**
-- **[- Dynamically modified the DOM and add basic events.]**
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-## :computer: Getting Started
+<!-- LIVE DEMO -->
 
-To get a local copy up and running, follow these steps.
+## 🚀 Live Demo <a name="live-demo"></a>
 
-### Prerequisites
+<ul>
+  <li><a href="https://github.com/tasnimchy96/to-do-list.github.io/dist/">Live Demo</a></li>
+  <li><a href="https://github.com/tasnimchy96/to-do-list.github.io/dist/"><img src="./Capture d'écran 2023-06-17 184926.png" width="50%"></a></li>
+</ul>
```

Comment:
[OPTIONAL] Please fix your live demo link because now it's not working so kindly fix it :+1: